upcoming_release_notes.md

```diff
@@ -0,0 +1,3 @@
+Thanks for trying out Firefox Daylight! In {version}, we have lots of new improvements:
```

Comment:
TODO for myself before landing: Tell the people in charge of releases about this file and how to update it. 

TODO as a followup: Let bump task clobber this file.

Reply:
For latter I've added a comment in https://bugzilla.mozilla.org/show_bug.cgi?id=1635488#c9 to track it when we add that support.